qcodes/tests/dataset/dond/test_do2d.py

```diff
@@ -154,9 +154,12 @@ def test_do2d_output_data(_param, _param_complex, _param_set, _param_set_2):
     )
     data = exp[0]
 
-    assert data.parameters == (
-        f"{_param_set.name},{_param_set_2.name}," f"{_param.name},{_param_complex.name}"
-    )
+    assert set(data.description.interdeps.names) == {
```

Comment:
as agreed offline, let's keep testing for the order here so that we have more chanced to know when it changed if it changed

Reply:
done